app/src/main/java/com/macbitsgoa/events/speakers/SpeakerCardFragment.java

```diff
@@ -0,0 +1,59 @@
+package com.macbitsgoa.events.speakers;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.google.android.material.card.MaterialCardView;
+import com.macbitsgoa.events.R;
+import com.macbitsgoa.events.home.HomeCardInterface;
+
+import androidx.fragment.app.Fragment;
+
+/*
+ * Created by Hariram.R
+ * 13/06/2018
+ * */
+
+public class SpeakerCardFragment extends Fragment implements View.OnClickListener,
+        HomeCardInterface {
+
+    private MaterialCardView cardView;
+
+
+    public static SpeakerCardFragment newInstance() {
```

Comment:
This method is unnecessary as no arguments are passed

Reply:
Check line 105 of home activity, it is being used there.

Reply:
it can be replaced simply by `new SpeakerCardFragment()` (Also, you don't have to write an empty constructor i.e. `public SpeakerCardFragment(){}`, the compiler does that automatically for you by default)